omise/omise.php

```diff
@@ -290,6 +335,30 @@ public function install()
         return true;
     }
 
+    /**
+     * A PrestaShop version specific version of 'display'
+     *
+     * @return see parent 'display' method
+     */
+    protected function versionSpecificDisplay($file, $template)
+    {
+        return $this->display($file, PRESTASHOP_VERSION_VIEW_PATH . $template);
+    }    
+
+    /**
+     * Register/Unregister all hooks for the module
+     *
+     * @return bool
+     */
+    protected function applyToHooks($callable)
+    {
+        $res = true;
+        foreach (explode(',', PRESTASHOP_PAYMENTMODULE_HOOKS) as $hook) {
+            if (!$res &= call_user_func($callable, $hook)) break;
```

Comment:
@jonrandy just wonder, can we just simply return `false` if the result of `call_user_func($callable, $hook))` is not `true`? (also, seems `if (!$res &= call_user_func($callable, $hook)) break;` converts the boolean result to `0`, `1`.)

Reply:
That's exactly what it does - I just prefer returns to be at the end of a function if possible, not in the middle. I've switched back to a boolean although not really necessary

Reply:
@jonrandy then I guess `@return bool` is a bit misleading as the return type is not exactly boolean. Can we update that?

Reply:
Re-read the above

Reply:
I often write code that takes advantage or truthiness and falsiness

Reply:
@jonrandy am I missing something? 🤔 
I was checking the type of the code at line 👉  `return $res;` and got `int(0)` and `int(1)`